src/frontend/src/flows/recovery/inputSeedPhrase.ts

```diff
@@ -13,27 +13,56 @@ const pageContent = () => html`
       font-size: 1rem;
       font-weight: 400;
     }
+
+    #warningBadAnchor {
+      color: red;
+      text-align: center;
+      width: 100%;
+    }
+
+    .full-width {
+      width: 100%;
+    }
   </style>
-  <div class="container">
+  <div class="container full-width">
     <h1>Your seed phrase</h1>
     <p>Please provide your seed phrase</p>
+    <p id="warningBadAnchor"></p>
     <textarea id="inputSeedPhrase" placeholder="Your seed phrase"></textarea>
     <button id="inputSeedPhraseContinue" class="primary">Continue</button>
     <button id="inputSeedPhraseCancel">Cancel</button>
   </div>
 `;
 
-export const inputSeedPhrase = async (): Promise<string | null> => {
+export const inputSeedPhrase = async (
+  userNumber: bigint
+): Promise<string | null> => {
   const container = document.getElementById("pageContent") as HTMLElement;
   render(pageContent(), container);
-  return init();
+  return init(userNumber);
 };
 
-const init = (): Promise<string | null> =>
+const init = (userNumber: bigint): Promise<string | null> =>
   new Promise((resolve) => {
     const inputSeedPhraseInput = document.getElementById(
       "inputSeedPhrase"
     ) as HTMLInputElement;
+    const warningBadAnchor = document.getElementById(
+      "warningBadAnchor"
+    ) as HTMLParagraphElement;
+    inputSeedPhraseInput.oninput = () => {
+      const lun = leadingUserNumber(inputSeedPhraseInput.value);
+      console.log(leadingUserNumber(inputSeedPhraseInput.value));
+      if (lun !== null && lun !== userNumber) {
+        warningBadAnchor.innerHTML = `<strong>warning</strong><br/>
+                    it looks like you are using a seed phrase for anchor <strong>${lun}</strong> but the anchor you are trying to recover is <strong>${userNumber}</strong>
+            `;
+
+        console.log("not ok...");
```

Comment:
Do you want to keep this `console.log`?  And if you keep it do you want `console.warn()` or similar instead?  If kept the text should probably say what is not ok. :-)